dist/formrenderer.js

```diff
@@ -1553,27 +1553,27 @@ window.JST["fields/address"] = function(__obj) {
       _print(_safe('\n\n'));
     
       if (format !== 'city_state' && format !== 'city_state_zip' && format !== 'country') {
-        _print(_safe('\n  <div class=\'fr_input_grid\'>\n    <div class=\'fr_item_full\'>\n      <input type="text"\n             id="'));
+        _print(_safe('\n  <div class=\'fr_input_grid\'>\n    <div class=\'fr_item_full\'>\n      <label class="fr_sub_label">Address</label>\n      <input type="text"\n             id="'));
```

Comment:
Unless I misunderstand, the label _is_ on top. Let's comment on the non-minified diff, it might be clearer there.